README.md

```diff
@@ -128,9 +128,10 @@ Since the local app uses OpenAI models, you should first deploy it for the optim
 1. Run these commands to install the web app as a local package (named `fastapi_app`), set up the local database, and seed it with test data:
 
     ```bash
+    python3 -m pip install -e src/backend
     python3 -m pip install -e src
-    python ./src/fastapi_app/setup_postgres_database.py
-    python ./src/fastapi_app/setup_postgres_seeddata.py
+    python ./src/backend/fastapi_app/setup_postgres_database.py
+    python ./src/backend/fastapi_app/setup_postgres_seeddata.py
```

Comment:
This now how it works but it's not pretty due to the issue I mentioned below.

vscode resolution will still work as is and running the launch.json scripts will work as is.